rasa/nlu/tokenizers/whitespace_tokenizer.py

```diff
@@ -11,19 +11,29 @@ class WhitespaceTokenizer(Tokenizer, Component):
 
     provides = ["tokens"]
 
+    defaults = {
+        # text will be tokenized with case sensitive as default
+        "case_sensitive": True
+    }
+
+    def __init__(self, component_config: Dict[Text, Any] = None) -> None:
+        """Construct a new tokenizer using the WhitespaceTokenizer framework."""
+
+        super(WhitespaceTokenizer, self).__init__(component_config)
+
```

Comment:
in the style of other components, let's create `self.case_sensitive = self.component_config["case_sensitive"]` in init

Reply:
sure makes sense, moved to init method